android/src/main/java/com/thebluealliance/androidclient/Utilities.java

```diff
@@ -1,31 +1,36 @@
 package com.thebluealliance.androidclient;
 
+import com.google.common.base.Predicate;
+
+import com.thebluealliance.androidclient.activities.GamedayActivity;
```

Comment:
Once we decide on the "proper" order and have everyone set it, yeah, that would be a good PR to do.